CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
```

Comment:
CMake 3.18 is needed to use CUDA with C++17. See https://cmake.org/cmake/help/latest/prop_tgt/CUDA_STANDARD.html